rideshare.rb

```diff
@@ -0,0 +1,149 @@
+# Ga-Young Jin, Cohort 12
+# Wednesday, August 7th, 2019
+# Week 1
+
+rideshare_data = [
+  [
+    { driver: "DR0001",
+      date: "3rd Feb 2016", 
```

Comment:
You have these arrays grouped by driver, but chose to leave the data inside these hashes, even though it's redundant. When I see groupings like this, it makes me think the outermost layer should probably be a hash!